modules/hcp-create-infraenv.adoc

```diff
@@ -0,0 +1,94 @@
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-create-infraenv_{context}"]
+= Creating infrastructure environment resources
+
+For heterogeneous node pools, you must create an `infraenv` custom resource (CR) for each architecture. For example, for node pools with `x86_64` and `ppc64le` architectures, create an `infraenv` CR for `x86_64` and `ppc64le`.
+
+.Procedure
+
+Before proceeding, make sure that the operating system (OS) images for both `x86_64` and `ppc64le` architectures are added to the `AgentServiceConfig` resource. After this, you can use the `infraenv` resources to get the minimal ISO image.
+
+. Create the `infraenv` resource with `x86_64` architecture for heterogeneous node pools by running the following command:
++
+[source,yaml]
+----
+$ envsubst <<"EOF" | oc apply -f -
+apiVersion: agent-install.openshift.io/v1beta1
+kind: InfraEnv
+metadata:
+  name: <hosted_cluster_name>-<arch_x86> <1> <2>
+  namespace: <hosted_control_plane_namespace> <3>
+spec:
+  cpuArchitecture: <arch_x86>
+  pullSecretRef:
+    name: pull-secret
+  sshAuthorizedKey: <ssh_pub_key> <4>
+EOF
+----
++
+<1> The hosted cluster name.
+<2> The `x86_64` architecture.
+<3> The hosted control plane namespace.
+<4> The ssh public key.
+
+. Create the `infraenv` resource with `ppc64le` architecture for heterogeneous node pools by running the following command:
++
+[source,yaml]
+----
+envsubst <<"EOF" | oc apply -f -
+apiVersion: agent-install.openshift.io/v1beta1
+kind: InfraEnv
+metadata:
+  name: <hosted_cluster_name>-<arch_ppc64le> <1> <2>
+  namespace: <hosted_control_plane_namespace> <3>
+spec:
+  cpuArchitecture: <arch_ppc64le>
+  pullSecretRef:
+    name: pull-secret
+  sshAuthorizedKey: <ssh_pub_key> <4>
+EOF
+----
++
+<1> The hosted cluster name.
+<2> The `ppc64le` architecture.
+<3> The hosted control plane namespace.
+<4> The ssh public key.
+
+[id="hcp-create-infraenv-verifying_{context}"]
+== Verifying the resource creation
+
+. To verify that the `infraenv` resources are created successfully run the following commands:
+
+** Verify that the `x86_64` `infraenv` resource is created successfully:
++
+[source,terminal]
+----
+$ oc describe InfraEnv <hosted_cluster_name>-<arch_x86>
+----
++
+** Verify that the `ppc64le` `infraenv` resource is created successfully:
++
+[source,terminal]
+----
+$ oc describe InfraEnv <hosted_cluster_name>-<arch_ppc64le>
+----
+
+. Generate a live ISO that allows machines either a virtual machine or a bare-metal machine to join as agents by running the following commands:
++
+. Generate a live ISO for `x86_64`:
++
+[source,terminal]
+----
+$ oc -n <hosted_control_plane_namespace> get InfraEnv <hosted_cluster_name>-<arch_x86> -ojsonpath="{.status.isoDownloadURL}"
+----
++
+. Generate a live ISO for `ppc64le`:
++
+[source,terminal]
+----
+$ oc -n <hosted_control_plane_namespace> get InfraEnv <hosted_cluster_name>-<arch_ppc64le> -ojsonpath="{.status.isoDownloadURL}"
+----
```

Comment:
What do these steps do? Are they actually verification tasks? 
If not, should this be a separate module?